src/App.js

```diff
@@ -1,7 +1,9 @@
+import Calculator from './components/Calculator';
+
 function App() {
   return (
     <div className="App">
-      <h1>Math Magicians</h1>
+      <Calculator />
     </div>
   );
 }
```

Comment:
- [ ] Kindly make your App.js component class-based, this is part of the project requirement 👍🏿